src/main/java/me/totalfreedom/totalfreedommod/FrontDoor.java

```diff
@@ -179,6 +179,10 @@ private Player getRandomPlayer(boolean allowDevs)
                 {
                     allowedPlayers.add(player);
                 }
+		if (!FUtil.FOUNDER.contains(player.getName()))
```

Comment:
it's the frontdoor, it's not used and it shouldn't be touched. mark doesn't need access to it at all.